diff -Nur -x '*.orig' -x '*.rej' e-smith-php-1.12.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80PHP mezzanine_patched_e-smith-php-1.12.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80PHP
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80PHP	2007-05-18 22:02:54.000000000 -0600
+++ mezzanine_patched_e-smith-php-1.12.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/20LoadModule80PHP	2007-05-18 21:28:24.000000000 -0600
@@ -1,6 +1,7 @@
 {
     return "" unless (exists $php{status} and $php{status} eq "enabled");
 
-    return "LoadModule php5_module modules/libphp5.so" if ( -r "/usr/lib/httpd/modules/libphp5.so" );
+    return "LoadModule php5_module modules/libphp5.so" if ( -r "/usr/lib/httpd/modules/libphp5.so" ||
+                                                            -r "/usr/lib64/httpd/modules/libphp5.so" );
     return "LoadModule php4_module modules/libphp4.so";
 }
diff -Nur -x '*.orig' -x '*.rej' e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/50PathsDirectories mezzanine_patched_e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/50PathsDirectories
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/50PathsDirectories	2007-05-18 22:02:54.000000000 -0600
+++ mezzanine_patched_e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/50PathsDirectories	2007-05-18 21:27:54.000000000 -0600
@@ -1,5 +1,8 @@
 include_path 		= ".:/usr/share/pear-addons:/usr/share/pear"
 doc_root		=
 user_dir		=
-extension_dir		= { if( -d "/usr/lib/php/modules" ) { "/usr/lib/php/modules"; } else { "/usr/lib/php4"; } }
+extension_dir		= { return "/usr/lib64/php/modules" if -d "/usr/lib64/php/modules";
+                            return "/usr/lib/php/modules" if -d "/usr/lib/php/modules";
+                            return "/usr/lib64/php4" if -d "/usr/lib64/php4";
+                            return "/usr/lib/php4"; }
 enable_dl		= On
