--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/20ResourceLimits.fixSpacing	2006-03-17 01:07:08.000000000 +0100
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/20ResourceLimits	2008-08-19 22:59:16.000000000 +0200
@@ -1,8 +1,8 @@
-max_execution_time = {
+max_execution_time                     = {
     my $et = $php{MaxExecutionTime} || "30";
     $OUT .= "$et";
 }
-memory_limit = {
+memory_limit                           = {
     my $ml = $php{MemoryLimit} || "32M";
     $OUT .= "$ml";
 }
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/60FileUploads.fixSpacing	2006-03-17 01:07:08.000000000 +0100
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/60FileUploads	2008-08-19 23:04:17.000000000 +0200
@@ -1,5 +1,5 @@
-file_uploads	= On
-upload_max_filesize = {
+file_uploads                           = On
+upload_max_filesize                    = {
     my $mf = $php{UploadMaxFilesize} || "10M";
     $OUT .= "$mf";
 }
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/10LanguageOptions.fixSpacing	2002-03-15 18:55:51.000000000 +0100
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/10LanguageOptions	2008-08-19 22:57:24.000000000 +0200
@@ -1,24 +1,24 @@
 {#;;;;;;;;;;;;;;;;;;
 # Language Options ;
 #;;;;;;;;;;;;;;;;;;;}
-engine			=	On
-short_open_tag	=	On	;
-asp_tags		=	Off
-precision		=	14
-y2k_compliance	=	Off
-output_buffering	= Off
-output_handler		=
-implicit_flush		= Off
-allow_call_time_pass_reference	= On
-safe_mode		=	Off
-safe_mode_exec_dir	=
-safe_mode_allowed_env_vars = PHP_
-safe_mode_protected_env_vars = LD_LIBRARY_PATH
-disable_functions	=
-highlight.string	=	#DD0000
-highlight.comment	=	#FF8000
-highlight.keyword	=	#007700
-highlight.bg		=	#FFFFFF
-highlight.default	=	#0000BB
-highlight.html		=	#000000
-expose_php	=	On
+engine                                 = On
+short_open_tag                         = On;
+asp_tags                               = Off
+precision                              = 14
+y2k_compliance                         = Off
+output_buffering                       = Off
+output_handler                         =
+implicit_flush                         = Off
+allow_call_time_pass_reference         = On
+safe_mode                              = Off
+safe_mode_exec_dir                     =
+safe_mode_allowed_env_vars             = PHP_
+safe_mode_protected_env_vars           = LD_LIBRARY_PATH
+disable_functions                      =
+highlight.string                       = #DD0000
+highlight.comment                      = #FF8000
+highlight.keyword                      = #007700
+highlight.bg                           = #FFFFFF
+highlight.default                      = #0000BB
+highlight.html                         = #000000
+expose_php                             = On
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/40DataHandling.fixSpacing	2006-03-17 01:07:08.000000000 +0100
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/40DataHandling	2008-08-19 23:01:57.000000000 +0200
@@ -1,15 +1,16 @@
-variables_order		=	"EGPCS"
-register_globals	=	Off
-register_argc_argv	=	On
-post_max_size		={
+mysql.allow_persistent                 = On
+variables_order                        = "EGPCS"
+register_globals                       = Off
+register_argc_argv                     = On
+post_max_size                          ={
     my $ms = $php{PostMaxSize} || "20M";
     $OUT .= "$ms";
 }
-gpc_order			=	"GPC"
-magic_quotes_gpc	=	On
-magic_quotes_runtime=	Off
-magic_quotes_sybase	=	Off
-auto_prepend_file	=
-auto_append_file	=
-default_mimetype = "text/html"
+gpc_order                              = "GPC"
+magic_quotes_gpc                       = On
+magic_quotes_runtime                   = Off
+magic_quotes_sybase                    = Off
+auto_prepend_file                      =
+auto_append_file                       =
+default_mimetype                       = "text/html"
 
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/50PathsDirectories.fixSpacing	2008-08-19 22:53:21.000000000 +0200
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/50PathsDirectories	2008-08-19 23:03:34.000000000 +0200
@@ -1,8 +1,9 @@
-include_path 		= ".:/usr/share/pear-addons:/usr/share/pear"
-doc_root		=
-user_dir		=
-extension_dir		= { return "/usr/lib64/php/modules" if -d "/usr/lib64/php/modules";
-                            return "/usr/lib/php/modules" if -d "/usr/lib/php/modules";
-                            return "/usr/lib64/php4" if -d "/usr/lib64/php4";
-                            return "/usr/lib/php4"; }
-enable_dl		= On
+mysql.allow_persistent                 = On
+include_path                           = ".:/usr/share/pear-addons:/usr/share/pear"
+doc_root                               =
+user_dir                               =
+extension_dir                          = { return "/usr/lib64/php/modules" if -d "/usr/lib64/php/modules";
+                                           return "/usr/lib/php/modules" if -d "/usr/lib/php/modules";
+                                           return "/usr/lib64/php4" if -d "/usr/lib64/php4";
+                                           return "/usr/lib/php4"; }
+enable_dl                              = On
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/30ErrorHandling.fixSpacing	2002-03-15 19:05:53.000000000 +0100
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/30ErrorHandling	2008-08-19 23:00:28.000000000 +0200
@@ -1,7 +1,8 @@
-error_reporting		= E_ALL & ~E_NOTICE
-display_errors		= Off
-display_startup_errors	= Off
-log_errors		= On
-error_log		= syslog
-track_errors		= Off
-warn_plus_overloading	= Off
+mysql.allow_persistent                 = On
+error_reporting                        = E_ALL & ~E_NOTICE
+display_errors                         = Off
+display_startup_errors                 = Off
+log_errors                             = On
+error_log                              = syslog
+track_errors                           = Off
+warn_plus_overloading                  = Off
--- e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/80ModuleSettings08MySQL.fixSpacing	2008-08-19 22:53:21.000000000 +0200
+++ e-smith-php-1.12.0/root/etc/e-smith/templates/etc/php.ini/80ModuleSettings08MySQL	2008-08-19 22:54:19.000000000 +0200
@@ -6,5 +6,5 @@
 mysql.default_socket                   =
 mysql.default_hosT                     =
 mysql.default_user                     =
-mysql.default_password	                =
+mysql.default_password	               =
 
